Scripts/script-isError.yml

```diff
@@ -0,0 +1,25 @@
+commonfields:
+  id: isError
+  version: -1
+name: isError
+script: |-
+  res = executeCommand("getEntry", {"id":args.entryId});
+  if (isError(res[0])) {
+      return 'yes';
+  } else {
+      return 'no';
+  }
+type: javascript
+tags:
+- Condition
+- Utility
+comment: Check whether a given entry is an error. Use ${lastCompletedTaskEntries.[0]}
+  to check the previous task entry.
+enabled: true
+args:
+- name: entryId
+  required: true
+  default: true
+  description: Entry to check
+scripttarget: 0
+releaseNotes: "New script to check whether a given entry is an error"
```

Comment:
No need for releaseNotes on new content (only for content that was not there in previous versions)